flips/20220111-execution-effort.md

```diff
@@ -0,0 +1,557 @@
+| GetStorageUsed             | function calls                           | Gets the storage used of an account                                                                         |
+| GetValue                   | the number of bytes read from a register | Get the value of a storage register                                                                         |
+| ProgramChecked             | function calls                           | Cadence callback after a program (transaction/script/contract code) was checked                             |
+| ProgramInterpreted         | function calls                           | Cadence callback after a program (transaction/script/contract code) was interpreted                         |
+| ProgramParsed              | function calls                           | Cadence callback after a program (transaction/script/contract code) was parsed                              |
+| ResolveLocation            | function calls                           | Called to resolve a location (most notably to resolve an import to a specific contract location)            |
+| RevokeEncodedAccountKey    | function calls                           | Revokes an account's key                                                                                    |
+| SetProgram                 | function calls                           | Caches an interpreted program (contract)                                                                    |
+| SetValue                   | the number of bytes saved to a register  | Set a value to a storage register                                                                           |
+| UpdateAccountContractCode  | function calls                           | Updates an account's contract                                                                               |
+| ValueExists                | function calls                           | Checks if a certain register exists                                                                         |
+| ValueDecoded               | function calls                           | Cadence callback after a value was decoded into a cadence type                                              |
+| ValueEncoded               | function calls                           | Cadence callback after a cadence type was encoded into a cadence json                                       |
+
+### Data analysis
+
+The large amount of different weights chosen was intentional. This allows for better insight into which weights should actually be used.
+
+The Following chapters describe how the linear model fitting was done due to a significant amount of outliers in the data and how certain weights were eliminated, because they were either to correlated with other weights, to small, or had a lot of noise.
```

Comment:
```suggestion
The Following chapters describe how the linear model fitting was done due to a significant amount of outliers in the data and how certain weights were eliminated, because they were either redundant (correlated with other weights), insignificant (had small or no impact on the correlation), or had a lot of noise.
```